ui/app/components/secret-version-menu.js

```diff
@@ -1,5 +1,17 @@
 import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { next } from '@ember/runloop';
 
 export default class SecretVersionMenu extends Component {
+  @service router;
```

Comment:
Just catching this now 🤦‍♀️ we probably don't need this injected anymore?

Reply:
Good catch! I'll remove it here.